### PR TITLE
Add release hook to format the changelog by labels

### PR DESCRIPTION
**Pull request by rafaelalmeidatk**
~~This should already work without vercel/release#173~~
edit: Now with the change to a CLI flag, this has to be merged only after the above PR

One thing to keep in mind is the fact that since the requests to GH aren't authenticated, there is a limit of 60 requests/hour. I tested and it might be better to send the GH connection that release already has to the hook, so the rate limit is way higher

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| buildDuration | 11s | 10.8s | -228ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.868 | 1.788 | -0.08 |
| / avg req/sec | 1338.46 | 1398.33 | ⚠️ +59.87 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.123 | 1.129 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 2226.46 | 2214.56 | -11.9 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| index.html gzip | 952 B | 952 B | ✓ |
| link.html gzip | 957 B | 957 B | ✓ |
| withRouter.html gzip | 944 B | 944 B | ✓ |
| Overall change | 2.85 kB | 2.85 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| buildDuration | 12s | 12.2s | ⚠️ +144ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: f7cea07ea23579a6058113f6eba20ab0f1ad5be8

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "types": "lerna run types --stream",
     "typescript": "lerna run typescript",
     "prepublish": "lerna run prepublish",
-    "publish-canary": "lerna version prerelease --preid canary --force-publish && release --pre",
+    "publish-canary": "lerna version prerelease --preid canary --force-publish && release --pre --skip-questions",
```
Comment (rafaelalmeidatk) [inline package.json:32]: I think this has to be changed somewhere else too?

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| buildDuration | 12.5s | 12.5s | ⚠️ +5ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.371 | 2.377 | ⚠️ +0.01 |
| / avg req/sec | 1054.59 | 1051.54 | -3.05 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.409 | 1.491 | ⚠️ +0.08 |
| /error-in-render avg req/sec | 1773.91 | 1676.54 | -97.37 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 959 B | 959 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| buildDuration | 13.5s | 13.2s | -326ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: 72eda2b9e3df2f87437fd374e9ad98d42d9dbd24

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| buildDuration | 13s | 12.8s | -153ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.205 | 2.172 | -0.03 |
| / avg req/sec | 1133.62 | 1150.75 | ⚠️ +17.13 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.465 | 1.513 | ⚠️ +0.05 |
| /error-in-render avg req/sec | 1706.86 | 1652.29 | -54.57 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 959 B | 959 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| buildDuration | 13.8s | 13.6s | -204ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js release-config | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: be8aa4874965c0c479557534faa1e967b840001b

**Pull request merged**